test/e2e/framework/test.go

```diff
@@ -36,4 +36,7 @@ type RunningServer interface {
 type KcpConfig struct {
 	Name string
 	Args []string
+
+	LogToConsole bool
```

Comment:
https://github.com/kcp-dev/kcp/pull/502 proposes to add test args to enable configuring running servers and controllers in process. Maybe rebase on that and add another arg for enabling logging to the console? Unless I'm missing something, I think it would be preferable to allow configuring the test run via cli args vs having to edit code.